src/main/scala/io/qbeast/spark/index/OTreeDataAnalyzer.scala

```diff
@@ -325,6 +326,7 @@ object DoublePassOTreeDataAnalyzer extends OTreeDataAnalyzer with Serializable {
       dataFrame: DataFrame,
       indexStatus: IndexStatus,
       isReplication: Boolean): (DataFrame, TableChanges) = {
+    logTrace(s"Begin: analyze for index ${indexStatus.revision}")
```

Comment:
Since you are calling the `indexStatus.revision` a lot, might be better to initialize a `val` in the method?

Reply:
Also, maybe is useful to print whether a new Revision is triggered or not... (here -> https://github.com/Qbeast-io/qbeast-spark/blob/b6c648180874926a0d673ed983d776145faf5937/src/main/scala/io/qbeast/spark/index/OTreeDataAnalyzer.scala#L346 )

Reply:
And one last comment on that line (I swear): shouldn't we output "`Begin: analyze for index with revision...`"? 

(same with other logs in the method)

Reply:
Will do all three.

> maybe is useful to print whether a new Revision is triggered or not...

I'm wondering whether this is debug level or info level. It feels like info level to me as it is relevant to see that a new revision is being created as part of regular execution, but I'm wondering if this is going to be printed to often, in which case, it debug might be more appropriate. What do you think?

Reply:
Depending on the data and the configuration, it could be triggered fewer times (for example: if the data contains all the space indexed in all appends, it would be triggered only once. Or if the user specifies a broader min/max range for the columns to index in `stats`...)

Thinking on a "persona" level, I believe the message is more useful for debugging (analyzing the output of the index) than for getting informed of the state of the process.  

So, I would say debug 😄

Reply:
I have addressed this, please check.